Tests/kaas/kaas-sonobuoy-tests/scs_k8s_conformance_tests/scs_0215_v1_robustness_features_test.go

```diff
@@ -0,0 +1,319 @@
+       for _, cm := range configMaps.Items {
+           if strings.Contains(cm.Name, "event-rate-limit") {
+               t.Logf("Found standalone EventRateLimit configuration in ConfigMap: %s", cm.Name)
+               return
+           }
+       }
+
+       t.Error("No EventRateLimit configuration found in production cluster")
+   })
+}
+
+func Test_scs_0215_apiPriorityAndFairness(t *testing.T) {
+   clientset, err := setupClientset()
+   if err != nil {
+       t.Fatalf("Failed to setup clientset: %v", err)
+   }
+
+   if isKindCluster(clientset) {
+       t.Skip("Running on kind cluster - skipping APF test")
+   }
+
+   t.Run("Check_APF_Configuration", func(t *testing.T) {
+       pods, err := clientset.CoreV1().Pods("kube-system").List(context.Background(), metav1.ListOptions{
+           LabelSelector: "component=kube-apiserver",
+       })
+       if err != nil || len(pods.Items) == 0 {
+           t.Fatal("Failed to find kube-apiserver pod")
+       }
+
+       for _, container := range pods.Items[0].Spec.Containers {
+           for _, arg := range container.Command {
+               if strings.Contains(arg, "--enable-priority-and-fairness=true") {
+                   t.Log("APF enabled via Command Line Flags")
+                   return
+               }
+           }
+       }
+
+       t.Error("API Priority and Fairness not enabled")
+   })
+}
+
+func Test_scs_0215_rateLimitValues(t *testing.T) {
+   clientset, err := setupClientset()
+   if err != nil {
+       t.Fatalf("Failed to setup clientset: %v", err)
+   }
+
+   if isKindCluster(clientset) {
+       t.Skip("Running on kind cluster - skipping rate limit values test")
+   }
+
+   t.Run("Check_Rate_Limit_Values", func(t *testing.T) {
+       expectedValues := map[string]string{
+           "qps":   "5000",
+           "burst": "20000",
+       }
+
+       configMaps, _ := clientset.CoreV1().ConfigMaps("kube-system").List(context.Background(), metav1.ListOptions{})
+       for _, cm := range configMaps.Items {
+           var config string
+           switch {
+           case strings.Contains(cm.Name, "event-rate-limit"):
+               config = cm.Data["config.yaml"]
+           case cm.Name == "admission-configuration":
+               config = cm.Data["eventratelimit.yaml"]
+           case cm.Name == "kube-apiserver":
+               config = cm.Data["config.yaml"]
+           }
+
+           if config != "" {
+               allFound := true
+               for k, v := range expectedValues {
+                   if !strings.Contains(config, fmt.Sprintf("%s: %s", k, v)) {
+                       allFound = false
+                       break
+                   }
+               }
+               if allFound {
+                   return
+               }
+           }
+       }
+
+       t.Error("Recommended rate limit values (qps: 5000, burst: 20000) not found")
```

Comment:
yes you are right here, I have overseen that it is just recommended. However, for that reason I exclude the test for now. Maybe some test for this could be added in the future if needed. The overall check for the presence of event rate limits is there.